apps/cookbook/src/app/examples/label-example/label-example.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="example-container">
+  <h1>Label</h1>
+
+  <div class="example-frame">
+    <h2>Kirby-label</h2>
+    <cookbook-label-example-item></cookbook-label-example-item>
+  </div>
+
+  <div class="example-frame">
+    <h2>Label</h2>
```

Comment:
Maybe best to make this structure match the showcase 1:1 like we try elsewhere.
```suggestion
    <h2>Other label types</h2>
    <h3>Form Field with label</h3>
```